src/main/java/org/reaktivity/nukleus/http/internal/util/HttpUtil.java

```diff
@@ -17,8 +17,24 @@
 
 import static java.lang.Character.toUpperCase;
 
+import org.agrona.DirectBuffer;
+
 public final class HttpUtil
 {
+    private static final byte ASCII_32 = 0x20;
+    private static final byte ASCII_34 = 0x22;
+    private static final byte ASCII_37 = 0x25;
+    private static final byte ASCII_60 = 0x3C;
+    private static final byte ASCII_62 = 0x3E;
+    private static final byte ASCII_92 = 0x5C;
+    private static final byte ASCII_94 = 0x5E;
+    private static final byte ASCII_96 = 0x60;
+    private static final byte ASCII_123 = 0x7B;
+    private static final byte ASCII_124 = 0x7C;
+    private static final byte ASCII_125 = 0x7D;
+    private static final byte ASCII_127 = 0x7F;
```

Comment:
Use descriptive names from `ASCII` chart instead of numbers, also constants for low range and high range masks.
```
    private static final long ASCII_LOW_RANGE_LONG_MASK = 0x2020_2020_2020_2020L;
    private static final long ASCII_HIGH_RANGE_LONG_MASK = -0x8080_8080_8080_8080L;
    private static final byte ASCII_LOW_RANGE_MASK = 0x20;
    private static final byte ASCII_HIGH_RANGE_MASK = -0x80;

    private static final byte ASCII_SPACE = 0x20;
    private static final byte ASCII_DOUBLE_QUOTES = 0x22;
    private static final byte ASCII_PERCENT_SIGN = 0x25;
    private static final byte ASCII_LESS_THAN = 0x3C;
    private static final byte ASCII_GREATER_THAN = 0x3E;
    private static final byte ASCII_BACKSLASH = 0x5C;
    private static final byte ASCII_CARET = 0x5E;
    private static final byte ASCII_GRAVE = 0x60;
    private static final byte ASCII_OPEN_BRACE = 0x7B;
    private static final byte ASCII_VERTICAL_BAR = 0x7C;
    private static final byte ASCII_CLOSE_BRACE = 0x7D;
    private static final byte ASCII_DELETE = 0x7F;
```